install/kubernetes/cilium/templates/cilium-ingress-service.yaml

```diff
@@ -24,6 +24,12 @@ spec:
     protocol: TCP
     nodePort: {{ .Values.ingressController.service.secureNodePort }}
   type: {{ .Values.ingressController.service.type }}
+  {{- if .Values.ingressController.service.loadBalancerClass }}
+  loadBalancerClass: {{ .Values.ingressController.service.loadBalancerClass }}
```

Comment:
IIRC, this attribute is only available in k8s 1.24+. I am not 100% sure if this will fail for lower k8s versions, if it's the case, we might need to add a check based on k8s version here.